### ISSUE TITLE
Generated python-flask code fails when `tags` contain spaces

### ISSUE THREAD

**Issue by alanbchristie** [swagger-api/swagger-codegen#6627]
##### Description

When I compile for `-l python-flask` and use spaces in my Swagger tags, e.g.:

```
tags:
- name: General components
```

The generated Python server code, when executed as advised in the generated README fails. i.e. if you run this:

```
python3 -m swagger_server
```

You get the following error:

```
Failed to add operation for POST [...]
[...]
connexion.exceptions.ResolverError: <ResolverError: Cannot resolve operationId "swagger_server.controllers.general components_controller.component_accumulator_post"! Import error was "No module named 'swagger_server.controllers.general components_controller'">
```

The module `swagger_server.controllers.general components_controller` does not exist because it is compiled as `swagger_server.controllers.general_components_controller` ... the space replaced by an underbar.

##### Swagger-codegen version

latest at time of writing (master/HEAD).

##### Swagger declaration file content or url

Except posted above. Code is proprietary and I cannot share ATM.

##### Command line used for generation

java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -i /<MY-PROJECT>/swagger/swagger.yml -l python-flask -o ../runner

##### Steps to reproduce

I suspect you can just try using tags with spaces.

##### Related issues/PRs

None

##### Suggest a fix/enhancement

The generated code has to know what the generator created. If the generator inserts an underbar the generated code has to know this or .. the generator has to generate an error, i.e. "Spaces in tags not supported".

For now I have removed the space and everything works.

**Comment by wing328** [comment]
@alanbchristie thanks for reporting the issue. I just work on something similar and will try to fix it over the weekend.

**Comment by wing328** [comment]
@alanbchristie I've fixed the issue via #6637. Please pull the latest master to give it a try or use the SNAPSHOT version mentioned in the README.
